comic-mono: init at 2020-12-28

an-empty-string opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

Comic Mono is a nice programming font that I use (currently building from a private overlay). There is no open packaging request issue (although I'd open one if I couldn't package it).

The PR also adds myself as a maintainer so the package will build.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable: (none are applicable, this package doesn't add anything testable)
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage) (N/A)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`) (N/A, no binary files)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes) (N/A)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

an-empty-string committed:
maintainers: add an-empty-string

an-empty-string committed:
comic-mono: init at 2020-12-28

legendofmiracles commented on pkgs/data/fonts/comic-mono/default.nix:
```diff
@@ -0,0 +1,35 @@
+    tar -z -f $downloadedFile --wildcards -x \*.ttf --one-top-level=$out/share/fonts
+
+    mkdir -p $out/etc/fonts/conf.d
+    cp ${./comic-mono-weight.conf} $out/etc/fonts/conf.d/30-comic-mono.conf
```
${} already copies it in the store, so we can link to save a few extra bytes

```suggestion
    ln -s ${./comic-mono-weight.conf} $out/etc/fonts/conf.d/30-comic-mono.conf
```

an-empty-string replied:
Thanks, fixed!

totoroot reviewed (approved):
This looks good to me and I'd like to see your contribution merged. Too bad the PR got buried.
@SuperSandro2000, could you please review and merge?

totoroot commented:
@an-empty-string It seems as if the commit adding you as a maintainer must be resolved.
Could you please rebase from nixpkgs/master? I use comic-mono on NixOS too, so if you don't mind can you please add me as a package maintainer as well?

SuperSandro2000 commented on maintainers/maintainer-list.nix:
```diff
@@ -13432,4 +13432,10 @@
     github = "PhilippWoelfel";
     githubId = 19400064;
   };
+  an-empty-string = {
```
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

an-empty-string commented:
@totoroot glad to see I'm not the only comic mono user here :) I added you as a maintainer.

@SuperSandro2000 thanks, I've rebased against master and put my maintainer entry in the right spot alphabetically, let me know if you need any other changes!
